### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by Skooma-Breath**
fixed some typos

don't think you can play voicelines with console commands ( at least i couldn't figure out how) so used 
tes3mp.playspeech instead

made function that adds force to explosion lol

### CODE REVIEW

File: plateWars.lua [hunk 1]
```diff
@@ -365,7 +406,11 @@ function plateWarsDefusedTimer(pid, cellDescription, uniqueIndex)
 end
 
 function plateWars.hasBomb(pid)
-    return plateWars.bomb.baseData.carrierPid == pid
+    -- return plateWars.bomb.baseData.carrierPid == pid
+    if inventoryHelper.containsItem(Players[pid].data.inventory, plateWars.bomb.refIds.inventoryItem) then
```
Comment (Skooma-Breath) [inline plateWars.lua]: put this back here for bomb testing purposes before i noticed the addbomb command

**Comment by Nkfree** [comment]
Thank you for the minor fixes, I was aware of the sound ids are incomplete but I missed the other stuff.

I'm not sure about being able to play those files using console commands or not, but in case it is possible it should be preferable to use the playSoundVP functions, because they offer more freedom like altering a volume.

I don't know whether other guys agreed on the 'applied force after explosion' feature, I personally don't find it neccessary as this is an effect that isn't neither related to Counter-Strike nor doesn't seem to add anything beneficial in regards to gameplay.
In CS if you are running away from bomb's explosion there's 3 scenarios:

1. You run away intact
2. You are hit by the explosion but you survive with less HP than you had
3. You die

I'm afraid that having additional force applied to players could result in removing scenario 2. - imagine you survive the bomb explosion with 10HP (OOF) and then it sends you flying to later fall to death. Also I'm unsure if this feature is of any effect in interiors, where the ceiling is lower. Interior cells might be fit for the arena style - from the game perspective and can be instanced - which is useful if there were multiple games going on in one server.

Also, the bomb force needs to be related to the guys being hit by the explosion, in the current state it affects all of the players, even those who did escape the bomb, which doesn't make sense to me.

It seems you put a lot of effort into it and I might be just seeing the downsides **while your feature could actually be fun**, so I'd say we should at least try it. It depends on the other guys though.

File: plateWars.lua (hunk 1 above)
Comment (Nkfree) [reply plateWars.lua]: I'd like to stick to the previous version of the function that doesn't iterate player's inventory. We could - for testing purposes - make the carrierPid a table and have the removeBomb function delete pid from that table instead.

**Comment by Skooma-Breath** [comment]
valid point with the fall damge, the distance from bomb it will effect you at and the strength of the blast are modifiable though so you can set it to be fairly weak force if you want.

but no worries, definitely not a critical thing to add lol

**Pull request closed without merging**